www/front_src/src/PollerWizard/pollerStep1/index.tsx

```diff
@@ -39,33 +78,189 @@ const FormPollerStepOne = ({
+            <TextField
+              fullWidth
+              required
+              error={getError('server_name')}
+              label={t(labelServerName)}
+              name="server_name"
+              value={stepOneFormData.server_name}
+              onBlur={handleBlur}
+              onChange={handleChange}
+            />
+            <TextField
+              fullWidth
+              required
+              error={getError('server_ip')}
+              label={t(labelServerIp)}
+              name="server_ip"
+              value={stepOneFormData.server_ip}
+              onBlur={handleBlur}
+              onChange={handleChange}
+            />
+            <TextField
+              fullWidth
+              required
+              error={getError('centreon_central_ip')}
+              label={t(labelCentreonCentralIp)}
+              name="centreon_central_ip"
+              value={stepOneFormData.centreon_central_ip}
+              onBlur={handleBlur}
+              onChange={handleChange}
+            />
+          </div>
+        ) : (
+          <div className={classes.form}>
+            {waitListOption && waitListOption.length !== 0 && (
+              <SelectField
+                fullWidth
+                label={t(labelSelectPollerIp)}
+                name="server_ip"
+                options={waitListOption}
+                selectedOptionId={stepOneFormData.server_ip}
+                onChange={handleChange}
+              />
+            )}
+            <TextField
+              fullWidth
+              required
+              error={getError('server_name')}
+              label={t(labelServerName)}
+              name="server_name"
+              value={stepOneFormData.server_name}
+              onBlur={handleBlur}
+              onChange={handleChange}
+            />
+            <TextField
+              fullWidth
+              required
+              error={getError('server_ip')}
+              label={t(labelServerIp)}
+              name="server_ip"
+              value={stepOneFormData.server_ip}
+              onBlur={handleBlur}
+              onChange={handleChange}
+            />
+            <TextField
+              fullWidth
+              required
+              error={getError('centreon_central_ip')}
+              label={t(labelCentreonCentralIp)}
+              name="centreon_central_ip"
+              value={stepOneFormData.centreon_central_ip}
+              onBlur={handleBlur}
+              onChange={handleChange}
+            />
+          </div>
+        )}
+        <div className={classes.formButton}>
+          <Button size="small" onClick={goToPreviousStep}>
+            {t(labelPrevious)}
+          </Button>
+          <Button
+            color="primary"
+            disabled={nextDisabled}
+            size="small"
+            type="submit"
+            variant="contained"
+          >
+            {t(labelNext)}
+          </Button>
+        </div>
```

Comment:
You can extract those previous, next and finish buttons into a component with his own logic.
You will be able to re-use it across poller wizard's components